CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Update propsTable to have unique selector for styles
```

Comment:
Because this is a bug fix it should be placed into a Fixed section

Reply:
Updated here https://github.com/cerner/terra-dev-site/pull/289/commits/b61c4c573507e5437d665b178ab06b286449cd22